docs/serving/knative-kubernetes-services.md

```diff
@@ -5,25 +5,36 @@ weight: 9
 type: "docs"
 ---
 
-This document describes what is running when running knative serving.
+This guide describes the components that are active when running Knative Serving.
```

Comment:
```suggestion
This guide describes the [Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/) that are active when running Knative Serving.
```

Reply:
Since K8s services are a particular kind of thing, let's use that terminology here instead of components.